XCTWebDriverAgentLib/Commands/FBCustomCommands.m

```diff
@@ -51,4 +52,11 @@ + (NSArray *)routes
   return FBResponseDictionaryWithOK();
 }
 
++ (id<FBResponsePayload>)handleHideKeyboard:(FBRouteRequest *)request
+{
+    FBXCTSession *session = (FBXCTSession *)[FBSession activeSession];
+    [[session.application.windows elementBoundByIndex:0] tap];
```

Comment:
I understand that, however you could check if keyboard is even visible before performing tap.

It is interesting it tap on (0, 0). I would expect it taps and the centre, anyway. :)